### PR TITLE
Story 25

### PR DESCRIPTION
**Pull request by randy-darsh**
Related Issue: #3 

Summary: Add functionality for admin to edit items information

Author(s): randy-darsh and big dawg Joel Lindow

### CODE REVIEW

**Review by JoelLindow — COMMENTED** [review]
I hope Sam changes the redirects. Looks good otherwise

File: app/controllers/admin/items_controller.rb [hunk 1]
```diff
@@ -13,14 +13,29 @@ def create
...
+    end
+  end
+
+  def edit
+    @item = Item.find(params[:id])
+  end
+
+  def update
+    @item = Item.find(params[:id])
+    if @item.update(item_params)
```
Comment (amhursh) [inline app/controllers/admin/items_controller.rb:27]: Should work. We can discuss a different redirect (the actual image page that was updated) but I kind of like this the way it is.

File: app/views/admin/items/_form.html.erb [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+  <%= f.label :category %>
   <%= f.select :category_id, Category.all.collect { |c| [ c.title, c.id ] }, include_blank: false %>
-<% end %>
\ No newline at end of file
+
+  <%= f.label :image %>
+  <%= f.text_field :image %>
+
+  <%= f.label :status %>
+  <%= f.select :status, ["active", "retired"] %>
```
Comment (amhursh) [inline app/views/admin/items/_form.html.erb:18]: This is nice an clean.

**Review by amhursh — APPROVED** [review]
Looks good. I will merge shortly.

**Pull request merged**